src/scripts.js

```diff
@@ -55,22 +56,18 @@ const getTravelerTrips = (tripsData) => {
   });
 };
 
-const loadPage = () => {
-  generateNameMessage();
-  generateMoneySpentMessage();
+const generatePage = () => {
+  generateWelcomeMessage();
+  // generateDestinationChoices();
   generatePastGrid();
   generateCurrentGrid();
   generateFutureGrid();
   generatePendingGrid();
 };
```

Comment:
I loved that you created a function to call all the other functions that needed to be called when the page loads. Great job!